style.css

```diff
@@ -51,11 +53,12 @@ a {
   display: flex;
   flex-flow: row nowrap;
   justify-content: center;
-  margin: 50px;
+  margin: 75px;
```

Comment:
Would it look better if the picture had even spacing above and below? if you add margin-top: 120px I thought it looked pretty balanced.